src/ObjectSchema.js

```diff
@@ -278,6 +278,24 @@ const ObjectSchema = ({ schema = initialState, ...options } = {}) => {
       return { valueOf, isFluentSchema, FLUENT_SCHEMA, _getState, extend }
     },
 
+    /**
+     * Returns an object schema with only a subset of keys provided
+     *
+     * @param properties a list of properties you want to keep
+     * @returns {ObjectSchema}
+     */
+    only: properties => {
+      return ObjectSchema({
```

Comment:
```suggestion
    only: properties => {
      properties = (typeof properties === 'string') ? [properties] : properties;
      return ObjectSchema({
```

I was thinking to allow for a single string or a list of properties but decided that just an array of properties may be simpler. Happy to add ^ if y'all prefer `S.object().only('single_field')`

Reply:
Array is good for me.